lib/output/utils.js

```diff
@@ -12,6 +12,50 @@ const implSymbol = Symbol("impl");
 const sameObjectCaches = Symbol("SameObject caches");
 const ctorRegistrySymbol = Symbol.for("[webidl2js]  constructor registry");
 
+const intrinsicConstructors = [
```

Comment:
I think we only need Array for now? Can we remove the rest (while maybe adding a comment explaining that we're only adding what we need)?